src/sync/sync_manager.cpp

```diff
@@ -261,17 +261,13 @@ bool SyncManager::client_should_reconnect_immediately() const noexcept
     return m_client_reconnect_mode == ReconnectMode::immediate;
 }
 
-void SyncManager::reconnect()
+void SyncManager::request_that_sessions_reconnect()
```

Comment:
Is this really an improvement? The new name is effectively "please ask the sessions to reconnect". I could maybe see using `reconnect_sessions` if there were some ambiguity about what was being reconnected, but `reconnect` by itself doesn't really seem unclear to me?